app/inventory/rank-select/component.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import SelectValues from 'hospitalrun/utils/select-values';
+import computed from 'ember-computed';
+
+export default Ember.Component.extend({
+  label: 'Rank',
+  rankOptions: [],
+  prompt: ' ',
+  class: 'col-sm-2 test-inv-rank',
+
+  options: computed('rankOptions', function() {
+    return SelectValues.selectValues(this.get('rankOptions'));
+  }),
+
+  init() {
+    this._super(...arguments);
+
+    // set available options
+    this.set('rankOptions', Ember.A(['A', 'B', 'C']));
```

Comment:
because of this options can not be overwrite. Also there is unfinished discussion with @stefanpenner 

https://github.com/HospitalRun/hospitalrun-frontend/pull/225#discussion-diff-46771310
https://github.com/HospitalRun/hospitalrun-frontend/pull/225#discussion_r46775205